src/NuGet.Core/NuGet.Common/Utility/XmlUtility.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Xml.Linq;
+using static NuGet.Shared.XmlUtility;
+
+namespace NuGet.Common
+{
+    [Obsolete("Use NuGet.Shared.XmlUtility instead.")]
```

Comment:
customers using our nupkg can't use `NuGet.Shared.XmlUtility`. It's internal to the assemblies referencing the file.